homeassistant/components/xiaomi_aqara/sensor.py

```diff
@@ -19,6 +19,7 @@
     "illumination": ["lm", None, DEVICE_CLASS_ILLUMINANCE],
     "lux": ["lx", None, DEVICE_CLASS_ILLUMINANCE],
     "pressure": ["hPa", None, DEVICE_CLASS_PRESSURE],
+    "bed_activity": ["μm", None, DEVICE_CLASS_PRESSURE],
```

Comment:
Is this sensor really measuring pressure?

Reply:
this is vibration sensor. 
https://www.aqara.com/en/vibration_sensor.html

Reply:
> Is this sensor really measuring pressure?

this is vibration sensor.
https://www.aqara.com/en/vibration_sensor.html

Reply:
Then we should not use `DEVICE_CLASS_PRESSURE`. We can replace that with `None`.

Reply:
> Then we should not use `DEVICE_CLASS_PRESSURE`. We can replace that with `None`.

done.